features/facebookImport/src/model/importers/utils/importer-util.js

```diff
@@ -21,17 +21,22 @@ async function readJSONFile(relativeFilePath, zipFile) {
     return readFullPathJSONFile(entry);
 }
 
+function decode(s) {
+    let d = new TextDecoder();
+    let a = s.split("").map((r) => r.charCodeAt());
+    return d.decode(new Uint8Array(a));
+}
+
 async function readFullPathJSONFile(entry) {
```

Comment:
I kind of feel that this is a proper example of the helpfulness of a unit test here...

Reply:
You mean for this specific function, right?

Reply:
yep

Reply:
This file is actually pretty well covered by tests, except for line 65. Only they're higher level tests; yesterday we could check all different alternatives with those tests, which consistently failed if there was something wrong.